packages/react-composites/src/composites/CallComposite/utils/Utils.ts

```diff
@@ -264,7 +264,7 @@ type GetCallCompositePageFunction = ((
     call: CallState | undefined,
     previousCall: CallState | undefined,
     transferCall?: CallState,
-    originCall?: CallState,
+    isReturningFromBreakoutRoom?: boolean,
```

Comment:
When does this prop get updated? And who updates this prop?

Reply:
This prop gets set to true when the returnFromBreakout function is called on this [line](https://github.com/Azure/communication-ui-library/pull/5584/files#diff-e45d7e6861410e5848963c2e06fdcd6931f885ebb719491f3559649b85b64015R1235)
and is set to false whenever the current call state is back to 'Connected' or 'Connecting' on this [line](https://github.com/Azure/communication-ui-library/pull/5584/files#diff-e45d7e6861410e5848963c2e06fdcd6931f885ebb719491f3559649b85b64015R291)